xAPI.md

```diff
@@ -2643,23 +2643,53 @@ the owner of that extension IRI explicitly states that a particular form of code
 
 ## 1.0 Requests
 
-### 1.1 Request Types
+xAPI tracking is done in the form of HTTP Requests from the Activity Provider (client) to the LRS (server).  This 
+specification offers guidance in some aspects of this communication.  Where no guidance is offered, it is 
+recommended that those implementing xAPI use current industry best practices.
+
+### 1.1 Request Methods
+
+xAPI supports the use of PUT, POST, GET, DELETE, and HEAD.  Use of other methods is beyond the scope of xAPI.  Each of the following sections details the traditional use of each method and how xAPI may differ slightly.  For full 
+implementation details, see each of the Resources (link) below.
 
 #### 1.1.1 HTTP PUT
 
+The HTTP PUT method is used to either create or update.  The enclosed entity is stored by the corresponding  
+Resource.  If it is new, it is created.  If there is a conflict, it is updated.  xAPI has a major exception to this 
+in that Statements are not allowed to be updated and thus there are rules around using PUT to update them.  Also in 
+xAPI, a PUT can only add a single Statement at a time.
+
+A major difference between PUT and POST is that PUT is idempotent, meaning if the method is executed more than  
```

Comment:
It may be worth noting that this doesn't mean POST can't be idempotent, just that with PUT you can assume it is, while with POST you may need additional information.